CONTRIBUTING.md

```diff
@@ -42,7 +39,7 @@ You can format code and imports by calling:
 
 ```
 black wagtailsharing
-isort --recursive wagtailsharing
+isort wagtailsharing
```

Comment:
`--recursive` was [removed in isort 5.0.0](https://timothycrosley.github.io/isort/CHANGELOG/#500-penny-july-4-2020).

Reply:
I think that one is already resolved thanks will.